### ISSUE TITLE
Wrong date format

### ISSUE THREAD

**Issue by DaFri-Nochiterov** [format-message/format-message#130]
Test code:
```js
let formatMessage = require("format-message")
formatMessage("{d, date, long}", {d: new Date()}, "ru-RU")
```
Expected Result: `25 июня 2017 г.`
Actual Result (Node v8.0): `2017 M06 25`

On Node v7.9.0 it gives wrong result too (but at least month is printed): `June 25, 2017`

**Comment by DaFri-Nochiterov** [comment]
https://github.com/nodejs/node/issues/13907#issuecomment-310887289

Confirmed that node not supports other languages but English. Can you please provide argument in `setup` function or check if code running on node and use other library (`moment`, `icu-loader`, etc)?

**Comment by TimothyGu** [comment]
> check if code running on node

To be clear, Node.js *does* support `Intl` for other languages -- just not by default. It's also possible for the user to point Node.js to a ICU data file with support for only *some* languages (e.g. Spanish), but not  others (e.g. Russian), though that's rare. Beware though, `icu-loader` starts a *new Node.js process on every method call* which may or may not be something you want.

**Comment by DaFri-Nochiterov** [comment]
Yeah, thanks for pointing this. I missed this line in `icu-loader` description. Actually, browsing source code of `icu-loader` opened small thing for me to fix problem with `full-icu` (problem: "but node should detect this package after I copied this file! Eh.. I broke something again...") - argument `--icu-data-dir`, so..
```
PS test node --icu-data-dir=./node_modules/full-icu
> let formatMessage = require("format-message")
undefined
> formatMessage("{s, date, long}", {s: new Date()}, "ru-RU")
'25 июня 2017 г.'
```
It works, yay! 🙃🎉 

Still don't closing issue, info about `full-icu` could be in docs to help others.

**Comment by TimothyGu** [comment]
> but node should detect this package after I copied this file!

The message is a bug on full-icu's side: autodetection has not been implemented in Node.js. See https://github.com/unicode-org/full-icu-npm/issues/6 and my PR to help clarify that: https://github.com/unicode-org/full-icu-npm/pull/15.

**Comment by thetalecrafter** [comment]
It sounds like all that needs to be done here is to document that node internationalization support requires special steps depending on the locales you want to use.

Would a note and a link to https://nodejs.org/api/intl.html be sufficient for this?
